### PR TITLE
AVRO-2678: Update node.js version (used for js tests) to current LTS (10.x.x)

### PR DESCRIPTION
**Pull request by iemejia**
R: @RyanSkraba

### CODE REVIEW

File: share/docker/Dockerfile [hunk 1]
```diff
@@ -69,7 +69,7 @@ RUN apt-get -qqy update \
  && rm -rf /var/lib/apt/lists
 
 # Install nodejs 6
-RUN curl -sSL https://deb.nodesource.com/setup_6.x \
+RUN curl -sSL https://deb.nodesource.com/node_10.x \
```
Comment (RyanSkraba) [inline share/docker/Dockerfile]:
```suggestion
RUN curl -sSL https://deb.nodesource.com/setup_10.x \
```
I think you meant this URL!  Comment to be updated...

**Comment by RyanSkraba** [comment]
I'm for this change in general -- node 6 is well out of support.  Node 12 is the latest LTS though, not 10.

This is used for more than just running the tests...  I'd love to hear from anyone with a bit more Javascript ecosystem experience to chime in whether this is safe!

File: share/docker/Dockerfile (hunk 1 above)
Comment (iemejia) [reply share/docker/Dockerfile]: thanks, done

**Comment by iemejia** [comment]
Actually I took version 10 to be more conservative than 12 (current/upcoming LTS). In any case we should not be running on unmaintained upstream versions, but we have of course to be conservative for users in old versions benefit.

**Comment by iemejia** [comment]
Thanks Ryan!

**Pull request merged**